src/components/Ui/Input.vue

```diff
@@ -39,42 +39,46 @@ onMounted(() => {
 </script>
 
 <template>
-  <div
-    class="border border-skin-border transition-colors bg-transparent text-skin-link rounded-3xl outline-none leading-[46px] text-left w-full mb-2 flex px-3 focus-within:border-skin-link hover:border-skin-link"
-    :class="{ '!border-red': error, 'cursor-pointer': $slots.selected }"
-  >
-    <div class="text-skin-text mr-2 whitespace-nowrap">
-      <slot name="label" />
-    </div>
+  <div class="min-h-[48px] rounded-3xl overflow-hidden">
     <div
-      v-if="$slots.selected"
-      class="flex-auto whitespace-nowrap overflow-x-auto"
-      :class="{ 'cursor-not-allowed text-skin-border': disabled }"
+      class="border border-skin-border transition-colors rounded-3xl outline-none leading-[46px] text-left w-full flex px-3 focus-within:border-skin-link hover:border-skin-link bg-skin-bg relative z-10"
+      :class="{ '!border-red': !!error }"
     >
-      <slot name="selected" />
+      <div class="text-skin-text mr-2 whitespace-nowrap">
+        <slot name="label" />
+      </div>
+      <div
+        v-if="$slots.selected"
+        class="flex-auto whitespace-nowrap overflow-x-auto"
+        :class="{ 'cursor-not-allowed text-skin-border': disabled }"
+      >
+        <slot name="selected" />
+      </div>
+      <input
+        v-else
+        :value="modelValue"
+        @input="handleInput"
+        ref="inputRef"
+        :placeholder="placeholder"
+        :type="number ? 'number' : 'text'"
+        :disabled="disabled"
+        class="input flex-auto w-full"
+        :class="[additionalInputClass, { 'cursor-not-allowed': disabled }]"
+        :required="required"
+        :maxlength="maxlength"
+        @blur="emit('blur')"
+      />
+      <slot name="info" />
     </div>
-    <input
-      v-else
-      :value="modelValue"
-      @input="handleInput"
-      ref="inputRef"
-      :placeholder="placeholder"
-      :type="number ? 'number' : 'text'"
-      :disabled="disabled"
-      class="input flex-auto w-full"
-      :class="[additionalInputClass, { 'cursor-not-allowed': disabled }]"
-      :required="required"
-      :maxlength="maxlength"
-    />
-    <slot name="info" />
-    <span
-      v-if="error"
-      v-tippy="{
-        content: error
-      }"
-      class="float-right text-skin-link"
+    <div
+      :class="[
+        'bg-red text-white text-sm relative z-0 transition-all flex items-center px-3 pb-2 pt-4 rounded-b-3xl',
```

Comment:
Can we use s-error class here?

Reply:
Yep. But I had to adjust the padding a bit. `s-error` used `pb-1`, changed it to `pb-2`, which suits both cases a bit better.

![image](https://user-images.githubusercontent.com/6792578/156569229-1474096a-06b7-421f-a045-e4f049f11730.png)
![image](https://user-images.githubusercontent.com/6792578/156569314-72227e49-3832-4aea-aaa2-a8352ce82672.png)

In the strategy settings the text seems to be a pixel to far at the top now. I guess that's because of the different heights/margins etc of the inputs. For me it looks ok like this.